dolphin/drivers/api.py

```diff
@@ -26,35 +27,50 @@ class API(object):
     def __init__(self):
         self.driver_manager = manager.DriverManager()
 
-    @staticmethod
-    def get_storage_registry():
-        """Show register parameters which the driver needs."""
-        pass
-
     def discover_storage(self, context, access_info):
```

Comment:
Function name may be not discover_storage considering what it is doing

Reply:
it's already in master branch, you can raise a issue to update it.

Reply:
Agree. It's just updating driver with storage info